app.rb

```diff
@@ -4,6 +4,7 @@
 require 'fluent/version'
 require 'fluent/log'
 require 'fluent/config'
+require 'fluent/plugin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.